### PR TITLE
Bump sqltoolsservice version in extensions

### PR DESCRIPTION
**Pull request by smartguest**
This is required to enable localization for STS, it adds the latest version of the sqltoolsservice to the extensions that use it.  I am testing this right now with the 1.30 build and will create a port request PR for that as well later.

### CODE REVIEW

**Comment by alanrenmsft** [comment]
@cssuh has another PR: https://github.com/microsoft/azuredatastudio/pull/15744, @cssuh ,may be you should close your PR? this one has more changes.

**Comment by alanrenmsft** [comment]
@smartguest you are updating the kusto extension, please include some one from Kusto team.

**Review by kburtram — APPROVED** [review]
I think there is already a pending PR with this same change.

**Comment by smartguest** [comment]
@kburtram this also adds it for Kusto as well, in addition to the changes in MSSQL.

**Comment by kburtram** [comment]
@smartguest please do get a review from Kusto extension owners before merging.

**Comment by smartguest** [comment]
@qifahs @JustinMDotNet @gupta1 Is it safe to update the sqltoolsservice for Kusto? This update is to add localization to the kusto extension primarily.

**Comment by Charles-Gagnon** [comment]
@smartguest It's a bit late to be getting this in for 1.30.0 - unless this is absolutely needed for some reason I would say you should plan on it just being in for 1.31.0.

**Comment by smartguest** [comment]
@Charles-Gagnon Yeah, I agree, additionally the change for calling the localization command is not in the 1.30 release as well, that would require another port request. Both of these features should be part of the 1.31 release for sure.

**Review by JustinMDotNet — APPROVED** [review]
Tested Kusto. Looks good.

**Pull request merged**
